### DISCUSSION TITLE
Interesting videos

### DISCUSSION THREAD

**Discussion by peterhinch** [micropython/micropython#10922] [General]
Odd that the creators don't post links to their videos, but these are worth a watch:
 1. [Damien on MP development](https://www.youtube.com/watch?v=FdZj5zUj5UE)
 2. [Matt Trentini February Melbourne meet-up](https://www.youtube.com/watch?v=bdfBH0k0AO0)

**Comment by mattytrentini** [comment]
Apologies, this is my fault; we do post them in discord and Twitter. I'm also many months behind keeping the [Melbourne MicroPython Blog](melbournemicropythonmeetup.github.io/) up-to-date but the idea is that is the 'official' archive...

I'll publish to the discussions board from now on! Give me a little while and I'll post links to all the recent videos in discussions too.
